pavelib/assets.py

```diff
@@ -384,14 +384,24 @@ def coffeescript_files():
 
 @task
 @no_help
-def compile_coffeescript(*files):
+def compile_coffeescript(*coffee_files):
     """
     Compile CoffeeScript to JavaScript.
     """
-    if not files:
-        files = ["`{}`".format(coffeescript_files())]
+    js_files = ""
+
+    if not coffee_files:
+        coffee_files = ["`{}`".format(coffeescript_files())]
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(coffeescript_files())
+    else:
+        js_files = r"`{} | sed 's/\.coffee/\.js/g'`".format(*coffee_files)
```

Comment:
Update: I'm abandoning this approach of adding a header to every coffeescript file in favor of just maintaining a complete list of the `.js` versions of them in the `.eslintignore`. It turns out to not be that many folders/files.